onboard-julia/form-field.tsx

```diff
@@ -0,0 +1,85 @@
+import React, {FC} from 'react';
+import {TextInputProps, TextProps} from 'react-native';
+import styled, {css} from 'styled-components/native';
+
+interface FormTextFieldProps extends TextInputProps {
+  error?: boolean;
+}
+
+interface FormTextProps extends TextProps {
+  error?: boolean;
+}
+
+const FormContainer = styled.View`
+  height: auto;
+  width: 300px;
+`;
+
+export const FormLabel = styled.Text<FormTextProps>`
+  font-size: 12px;
+  font-weight: normal;
+  color: #777777;
+  margin-bottom: 4px;
+  margin-top: 8px;
+
+  ${props =>
+    props.error &&
+    css`
+      color: red;
+    `}
+`;
+
+const FormTextField = styled.TextInput<FormTextFieldProps>`
+  border-width: 1px;
+  border-color: #777777;
+  padding: 8px;
+  margin-bottom: 8px;
+  width: 300px;
+  ${props =>
+    props.error &&
+    css`
+      border-color: red;
+    `}
+`;
+
+const FormCaption = styled.Text<FormTextProps>`
+  font-size: 12px;
+  font-weight: normal;
+  color: red;
+  margin-bottom: 8px;
+  text-align: justify;
+`;
+
+interface FormFieldProps {
+  label: string;
+  value: string;
+  onChangeText: (text: string) => void;
+  error: boolean;
+  captionText: string;
+  autoCapitalize?: 'none' | 'sentences' | 'words' | 'characters';
+  secureTextEntry?: boolean;
+}
+
+export const FormField: FC<FormFieldProps> = ({
+  label,
+  value,
+  onChangeText,
+  error,
+  captionText,
```

Comment:
Para ficar mais claro que as duas coisas estão relacionadas, usaria o mesmo nome para as duas props
```suggestion
  error,
  errorText,
```